enterprise/models.py

```diff
@@ -2733,13 +2733,36 @@ def __repr__(self):
         """
         return self.__str__()
 
+    @classmethod
+    def validate_compression(cls, enable_compression, data_type, delivery_method):
```

Comment:
Add a separate test for this method and pass different scenarios using `ddt`.

Reply:
Already a test case exists in test_models.py for this method.